treeherder/seta/analyze_failures.py

```diff
@@ -24,6 +25,7 @@ def __init__(self, **options):
 
     def run(self):
         fixed_by_commit_jobs = get_failures_fixed_by_commit()
+        load_preseed()
```

Comment:
Ah, ok. Another suggestion I can think of is to remove load_preseed() from treeherder's load_initial_data management command.

Instead we could add a seta specific management command (https://github.com/mozilla/treeherder/tree/master/treeherder/seta/management) called 'initialize_seta'' or 'reset_seta'. That command would:

- blow away any existing records in the seta JobPriority table
- run analyze_failures
- run load_preseed

And we'll just have to document that you need to run that command on a clean dev environment. It's risky that if someone runs that on production though they would empty JobPriorities. Maybe we can add a confirmation prompt "This will erase any existing Job Priority DB entries, are you sure?" or something like that.

That's maybe not the best solution but maybe at least reduces it to one step.